nextflow.config

```diff
@@ -19,3 +21,9 @@ workDir = 'work'
 docker {
     enabled = true
 }
+
+executor {
+    name = 'local'
```

Comment:
I think `local` is the default so no need to set this https://www.nextflow.io/docs/latest/reference/config.html#executor